Assets/Scripts/MarchingCubes/Planet.cs

```diff
@@ -55,6 +61,8 @@ public void Initialize(Transform player, int randomSeed, bool spawn)
 
         MinMaxTerrainLevel terrainLevel = new MinMaxTerrainLevel();
 
+        rotationAxis = RandomPointOnSphereEdge(radius, rand) - Vector3.zero;
```

Comment:
Here you could maybe use the built in function rand.OnUnitSphere() or rand.OnUnitSphereUniform()
The code for this would be something like the following:

```cs
// Either
rotationAxis = rand.OnUnitSphere() * radius;
// Or
rotationAxis = rand.OnUnitSphereUniform() * radius;
```

Otherwise I think everything looks good!

Reply:
This is now fixed

Reply:
👍